link-hint.el

```diff
@@ -1076,6 +1078,23 @@ Only search the range between just after the point and BOUND."
   :at-point-p #'xref--item-at-point
   :copy #'link-hint--copy-xref-item)
 
+;; ** Dictionary button
+(defun link-hint--next-dictionary-button (&optional end-bound start-bound)
+  (let ((pos (next-single-char-property-change (or start-bound (point))
+                                               'button
+                                               nil
+                                               end-bound)))
+    (when (< pos end-bound)
+      (if-let ((button (button-at pos)))
+          (overlay-start button)
+        (link-hint--next-dictionary-button end-bound pos)))))
+
+(link-hint-define-type 'dictionary-button
+  :next #'link-hint--next-dictionary-button
+  :vars '(dictionary-mode)
```

Comment:
Also add dictionary-mode to the generic button type's :not-vars.

Reply:
I've added both dictionary and Man mode, as it turned out that Man mode also used the exact same overlay button code as Woman mode.  I've consolidated Woman, Man, and dictionary types into a single 'overlay button' type in the commits I've just pushed.  Let me know if you think that's a good idea or whether I should separate them back out.